azure-pipelines.yml

```diff
@@ -34,11 +29,12 @@ resources:
 
 variables:
   # Container registry service connection established during pipeline creation
-  dockerRegistryServiceConnection: 'bf604a33-e9e3-4da3-b4ba-c395cc31b213'
+  dockerRegistryServiceConnection: '0fd848f2-70dd-4f38-a3aa-fd52c45e2423'
   imageRepository: 'georgeracuesrsbackend'
   containerRegistry: 'esrsproject.azurecr.io'
   dockerfilePath: '$(Build.SourcesDirectory)/Dockerfile'
   tag: '$(Build.BuildId)'
+  imagePullSecret: 'esrsproject212494a2-auth'
```

Comment:
Should this be in production code?

Reply:
This is just the name of the secret, not the value.